package cap11;

public class TesteErro {

	/**
	 * @param args
	 */
	public static void main(String[] args) {
		System.out.println("Inicio do main");
		try {
			metodo1();
		} catch (ArrayIndexOutOfBoundsException e){
			//e.printStackTrace();
			System.out.println("Exceção");
		}
		System.out.println("Fim do main");
	}

	private static void metodo1() {
		System.out.println("Inicio do metodo1");
		metodo2();
		System.out.println("Fim do metodo1");
	}

	private static void metodo2() {
		System.out.println("Inicio do metodo2");
		int[] array = new int[10];
		for (int i = 0; i <= 12; i++) {
			array[i] = i;
			System.out.println(i);
		}
		System.out.println("Fim  do metodo 2");
	}

}
